autopr/__init__.py

```diff
@@ -210,6 +210,59 @@ def reset():
     click.secho("Repositories marked as not done")
 
 
+def _print_repository_list(
+    title: str, repositories: List[database.Repository], total: int
+):
+    if len(repositories) > 0:
+        click.secho(f"{title} [{len(repositories)}/{total}]:", bold=True)
```

Comment:
What about moving this outside the conditional so we always print the "full" picture?

Reply:
Yeah that'd probably be better. I'll merge the other PR into this one and then take the code out of the conditional.